controllers/vcdmachine_controller.go

```diff
@@ -328,16 +335,16 @@ func checkIfMachineNodeIsUnhealthy(machine *clusterv1.Machine) bool {
 
 func (r *VCDMachineReconciler) reconcileCloudInitScript(ctx context.Context, vcdClient *vcdsdk.Client,
```

Comment:
Renamed the function to `reconcileNodeSetupScripts`.
The bootstrap script reconciliation is really simple, so for now I think it can stay in the same function. In case it gets more complicated, we can separate it.